pkg/cri/config/config.go

```diff
@@ -194,10 +193,6 @@ type Registry struct {
 	// The key is the domain name or IP of the registry.
 	// This option will be fully deprecated for ConfigPath in the future.
 	Configs map[string]RegistryConfig `toml:"configs" json:"configs"`
-	// Auths are registry endpoint to auth config mapping. The registry endpoint must
-	// be a valid url with host specified.
-	// DEPRECATED: Use ConfigPath instead. Remove in containerd 1.6.
-	Auths map[string]AuthConfig `toml:"auths" json:"auths"`
```

Comment:
why are we removing this? 

This doesn't not look trivial as correcting import order or typos. If we need to remove this can we separate it into a separate PR?